src/Xamarin.Android.Build.Tasks/Microsoft.Android.Sdk/targets/Microsoft.Android.Sdk.AssemblyResolution.targets

```diff
@@ -168,6 +169,27 @@ _ResolveAssemblies MSBuild target.
         IncludeDebugSymbols="$(AndroidIncludeDebugSymbols)">
       <Output TaskParameter="OutputLibraries" ItemName="FrameworkNativeLibrary" />
     </ProcessNativeLibraries>
+    <ItemGroup>
+      <LocalFrameworkNativeLibrary Include="$(IntermediateOutputPath)native\%(FrameworkNativeLibrary.RuntimeIdentifier)\%(FrameworkNativeLibrary.Filename)%(FrameworkNativeLibrary.Extension)">
+        <RuntimeIdentifier>%(FrameworkNativeLibrary.RuntimeIdentifier)</RuntimeIdentifier>
+      </LocalFrameworkNativeLibrary>
+    </ItemGroup>
+  </Target>
+
+  <Target Name="_StripFrameworkNativeLibraries"
+      Condition=" '$(AndroidIncludeDebugSymbols)' != 'true' "
+      Inputs="@(FrameworkNativeLibrary)"
+      Outputs="@(LocalFrameworkNativeLibrary)">
+    <StripNativeLibraries
+        Libraries="@(FrameworkNativeLibrary)"
+        LocalPath="$(IntermediateOutputPath)native"
+        AndroidBinUtilsDirectory="$(AndroidBinUtilsDirectory)">
+      <Output TaskParameter="OutputLibraries" ItemName="StrippedFrameworkNativeLibrary" />
```

Comment:
I *think* you *also* want to add `@(StrippedFrameworkNativeLibrary)` to `@(FileWrites)`, so that the "msbuild intermediate clean" doesn't *delete* these files when you don't expect them to be deleted.

@dellis1972? @jonathanpeppers?